app/Events/ImageUploaded.php

```diff
@@ -17,7 +17,7 @@ class ImageUploaded implements ShouldBroadcast, ShouldQueue
     public $city, $state, $country, $countryCode, $imageName, $teamName, $isUserVerified;
 
     // For CheckContributors
-    public $userId, $countryId, $stateId, $cityId, $teamId;
+    public $userId, $countryId, $stateId, $cityId, $latitude, $longitude, $teamId;
```

Comment:
This is for the zoom to location Easter Egg. Let's keep it a secret from our power-users for as long as we can, lol :D